src/plasmapy/particles/atomic.py

```diff
@@ -1159,7 +1159,8 @@ def _get_relevant_stopping_information(
             raise NotImplementedError(
                 "Stopping calculations for electrons have not been implemented yet!"
             )
-        elif incident_particle == Particle("H+"):
+        # TODO: is there a difference between H+ and p+ ??
```

Comment:
I don't think so, as long as you take the convention that H+ is the most common isotope with no neutrons, they should be the same? I think @namurphy mentioned some subtlety here to me at some point...

Reply:
For some reason it wasn't catching p+ in the if statement

Reply:
There is indeed a difference between `'H+'` and `'p+'`.  When we use `'H+'`, it uses the standard atomic weight of hydrogen (mostly H-1, but some D), which is how it's handled for all other elements.

This is probably one of the most common mix-ups 😕, so I'm wondering if we should have a warning when `Particle` is provided with `'H+'`...or at the very least, better note this in the documentation.

Reply:
What if we just check the charge and mass of the particle that is passed in?

Reply:
Another possibility would be `incident_particle in ParticleList(["H+", "p+"])`.

Reply:
I think Nick's solution is more Plasmapythonic :)